scripts/release.sh

```diff
@@ -174,10 +178,26 @@ function run_validation_tests() {
 # Initialize everything (flags, workspace, etc) for a release.
 function initialize() {
   parse_flags $@
+
+  # Log what will be done and where.
+  banner "Release configuration"
   echo "- Destination GCR: ${KO_DOCKER_REPO}"
+  (( SKIP_TESTS )) && echo "- Tests will NOT be run" || echo "- Tests will be run"
+  if (( TAG_RELEASE )); then
+    echo "- Artifacts will tagged '${TAG}'"
+  else
+    echo "- Artifacts WILL NOT be tagged"
+  fi
   if (( PUBLISH_RELEASE )); then
-    echo "- Destination GCS: ${RELEASE_GCS_BUCKET}"
+    echo "- Release WILL BE published to '${RELEASE_GCS_BUCKET}'"
+  else
+    echo "- Release will not be published"
   fi
+  if (( BRANCH_RELEASE )); then
+    echo "- Release WILL BE branched from '${RELEASE_BRANCH}'"
```

Comment:
can we add this to the next if condition for branch_release.

Reply:
I don't want to, since these are two distinct functional blocks: one for logging purposes only, the other for actions.

Reply:
okies.sg.